src/server.js

```diff
@@ -30,6 +30,22 @@ app.get('/', function(req, res) {
     res.redirect('/games');
 });
 
+app.get('/titles', function(req, res) {
+  output = [];
+  const fs = require('fs');
+  fs.readdir(__dirname + '/../zcode/', (err, files) => {
+    files.forEach(file => {
+      if (!fs.statSync(__dirname + '/../zcode/' + file).isDirectory()){
```

Comment:
It would probably be good to explicitly filter anything that isn't a ```.z?``` file, in case there are dotfiles or some other randomness lying around that directory.